crates/query-engine/sql/src/sql/ast.rs

```diff
@@ -108,6 +108,11 @@ pub enum From {
         alias: TableAlias,
         column: ColumnAlias,
     },
+    Unnest {
```

Comment:
Used for fancy queries like `in <variable>` or `in <column>`, which are converted to a subquery of unnest.